### PR TITLE
Missing info component

### PR DESCRIPTION
**Pull request by pedroespindula**
**Feature/Bug description:** Shown missing info doesn't match design.

**Solution:** Create is-empty-card component to be shown when something is empty.
- Design:
screenshot from 2019-02-07 18-25-03

- Before:
1. Timeline:
localhost_8081_ nexus 5 2

2. Notifications:
localhost_8081_notifications nexus 5 3

3. Events:
localhost_8081_ nexus 5 3

- After:
1. Timeline:
localhost_8081_notifications nexus 5

2. Notifications:
localhost_8081_notifications nexus 5 1

3. Events:
localhost_8081_notifications nexus 5 2

**TODO/FIXME:** n/a

### CODE REVIEW

File: frontend/notification/notifications_list_mobile.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <md-content>
-    <h3 id="mobile-notification-empty" ng-if="notificationListCtrl.notifications.length === 0">
-        <b>Nenhuma notificação no momento</b>
-    </h3>
+    <is-empty-card ng-if="notificationListCtrl.notifications.length === 0"
```
Comment (JuliePessoa) [inline frontend/notification/notifications_list_mobile.html]: Is this class 'mobile-notification-empty' being used in other place?

File: frontend/post/timeline.html [hunk 2]
```diff
@@ -1,10 +1,7 @@
 <!-- TIMELINE -->
 <div flex ng-if="!timelineCtrl.isLoadingPosts">
-  <md-card ng-if="timelineCtrl.posts.size() === 0">
-    <md-content layout-align="center center" layout="column">
-      <h4>Esta instituição ainda não possui publicações.</h4>
-    </md-content>
-  </md-card>
+  <is-empty-card ng-if="timelineCtrl.posts.size() === 0" text="Ainda não há publicações para serem mostradas">
```
Comment (JuliePessoa) [inline frontend/post/timeline.html]: Keep the same message

File: frontend/utils/is-empty-card/is-empty-card.component.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        templateUrl: "app/utils/is-empty-card/is-empty-card.html",
+        controller: isEmptyCardController,
+        controllerAs: "isEmptyCardCtrl",
+        bindings: {
+            text: "@"
+        },
+    });
+
+    function isEmptyCardController() {
+        const isEmptyCardCtrl = this;
```
Comment (Luiz-FS) [inline frontend/utils/is-empty-card/is-empty-card.component.js]: Remove the unused variable.

File: frontend/utils/is-empty-card/is-empty-card.component.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+(function () {
+    'use strict';
+
+    angular.module("app").component("isEmptyCard", {
+        templateUrl: "app/utils/is-empty-card/is-empty-card.html",
```
Comment (JuliePessoa) [inline frontend/utils/is-empty-card/is-empty-card.component.js]: Jsdoc, component and bindings

**Pull request merged**